apps/voting-disputable/test/contracts/voting_disputable.js

```diff
@@ -24,7 +24,7 @@ contract('Voting disputable', ([_, owner, representative, voter10, voter20, vote
     collateralToken = await agreementDeployer.deployCollateralToken()
     votingDeployer.previousDeploy = agreementDeployer.previousDeploy
 
-    await agreement.sign(voter40)
+    await agreement.sign({ from: voter40 })
```

Comment:
I'm wondering if we should primarily use a mock Agreement for these tests, and leave tests with the full Agreement as an integration test?

In any case, this will probably break once we merge with master since `sign()`'s signature has changed.

Reply:
Let me merge and explore this on top of the buidler changes :)